pepper-apis/src/main/java/org/broadinstitute/ddp/json/activity/ActivityInstanceSummary.java

```diff
@@ -55,6 +55,9 @@
     @SerializedName("isFollowup")
     private boolean isFollowup;
 
+    @SerializedName("prevInstanceGuid")
```

Comment:
Since this will show up in JSON response, how about we rename it to `previousInstanceGuid`? Also, can you update the docs in `pepper-apis/docs/specification`? Thanks!

Reply:
Property renamed, doc schema updated